### ISSUE TITLE
ui5-segmentedbutton: the method of interaction for segmented button not clear for screen reader users

### ISSUE THREAD

**Issue by BennyHuang** [SAP/ui5-webcomponents#2934]
**Describe the bug**
The method of interaction for segmented button not clear for screen reader users

**To reproduce**
Open JAWS and focus on the segmented button. The the instruction text announced by JAWS is not clear enough.

**Expected behavior**
The expected texts for screen reader should be same as what we have for sap.m.SegmentedButton
![image](https://user-images.githubusercontent.com/1995447/110690418-c41a4480-8198-11eb-882d-6b6015f5afd3.png)

**Context**
UI5 Web Components version: 0.29.6

**Affected components (if known)**
ui5-segmentedbutton

**Organization: (if applicable)**
SAP SuccessFactors

**Priority: (optional) (Low, Medium, High or Very High)**
Medium

**Comment by PetyaMarkovaBogdanova** [comment]
Dear colleagues from @SAP/ui5-webcomponents-topic-b,
can you please take a look at this issue.
Now only the selected toggle button is being read out.

Best regards,
Petya.

**Comment by ilhan007** [comment]
Hello @BennyHuang 

as mentioned in my previous message, fixing the announcements of the SegmentedButton would require significant refactoring. For that, we created a new issue: https://github.com/SAP/ui5-webcomponents/issues/3191

I will close the current one, it's is referenced in the new one for traceability. 
You can see the details and follow the progress here:  https://github.com/SAP/ui5-webcomponents/issues/3191

Best,
ilhan
